core/trino-spi/pom.xml

```diff
@@ -769,6 +769,17 @@
                                     <old>method io.trino.spi.connector.CatalogHandle.CatalogVersion io.trino.spi.eventlistener.QueryOutputMetadata::getCatalogVersion()</old>
                                     <new>method io.trino.spi.connector.CatalogVersion io.trino.spi.eventlistener.QueryOutputMetadata::getCatalogVersion()</new>
                                 </item>
+                                <item>
+                                    <ignore>true</ignore>
```

Comment:
These `<ignore>true</ignore>` are all redundant since it is set at the top level